dictionary.txt

```diff
@@ -867,6 +867,12 @@ _j	ⱼ
 \llblacktriangle	◣	
 \ulblacktriangle	◤	
 \urblacktriangle	◥	
+\bigblacktriangledown	▼ 
+\bigblacktriangleup	▲	
+\blacktriangle		▴
+\blacktriangledown	▾ 
+\blacktriangleleft	◀ 
+\blacktriangleright	▶ 
```

Comment:
The spaces are not correct

There should be 1 tab in between and 1 tab after as seen in the other examples

![image](https://user-images.githubusercontent.com/20955511/135603456-f26aa7c4-f656-4137-ac02-d12c2e99b139.png)


```suggestion
\bigblacktriangledown	▼	
\bigblacktriangleup	▲	
\blacktriangle	▴	
\blacktriangledown	▾	
\blacktriangleleft	◀	
\blacktriangleright	▶	
```